common.gypi

```diff
@@ -369,7 +369,7 @@
         'ldflags': [ '-pthread' ],
       }],
       [ 'OS in "linux freebsd openbsd solaris android aix cloudabi"', {
-        'cflags': [ '-Wall', '-Wextra', '-Wno-unused-parameter', ],
+        'cflags': [ '-Wall', '-Wextra', '-Wno-unused-parameter', '-std=c11', ],
```

Comment:
Since `common.gypi` is used by addons we should at least run this past citgm.

Reply:
I suppose an alternative is to hard-code the flag into the Makefile?

Reply:
Or maybe in the `binding.gyp` of the n-api benchmark addon?

Reply:
No need for CITGM. It's failing to build on lots of hosts.  Will force-push that commit out....